### DISCUSSION TITLE
Should you add your name to the description?

### DISCUSSION THREAD

**Discussion by NoobTracker** [ScratchAddons/ScratchAddons#160] [Help]
Currently, only the deleted comments thing includes "by abc" (in this case @JeffaloBob) in the description, so I wonder if you should always add that, if we should add a "creator" tag in the JSON that would be placed next to the name of the addon or just let it be.

**Comment by Explosion-Scratch** [comment]
That sounds good! It would be nice to credit everyone about this.

**Comment by Hans5958** [comment]
griffpatch's and WorldLanguages' also have one.

I don't mind about having the creator names on it, since in later cases the addons are going to be modified multiple times, so it's not going to be just a single creator. But, if this is got implemented, other than using the word `author` instead of the word `creator`, please also add the `contributors` field for other contributors on an addon.

**Comment by NoobTracker** [comment]
Mockup time!
![credits](https://user-images.githubusercontent.com/63962365/91979130-91911a00-ed25-11ea-8bde-f562f46ea8a8.PNG)
Yep, `author`, `creator`, whatever. `author `really sounds better.

**Comment by jeffalo** [comment]
i think an author key in each add-on manifest would be cool

**Comment by WorldLanguages** [comment]
Not all addons have author names, so having a special manifest key for those sounds weird. Imo they should stay at the end of the description.

**Comment by NoobTracker** [comment]
You could add those keys and if there is none, there is none.
